<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Добавление</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="labelDateAdd">
            <property name="text">
             <string>Дата добавления</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDateEdit" name="dateAdd">
            <property name="date">
             <date>
              <year>2001</year>
              <month>1</month>
              <day>1</day>
             </date>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelDistrict">
            <property name="text">
             <string>Район</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="district"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelIdNumber">
            <property name="text">
             <string>Номер удостоверения</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="idNumber"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelName">
            <property name="text">
             <string>ФИО</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="name"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelBirthDate">
            <property name="text">
             <string>День рождения</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="birthDate">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="labelAddress">
            <property name="text">
             <string>Адрес</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="address"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelPhoneNumber">
            <property name="text">
             <string>Телефон</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="phoneNumber"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelDiabetesType">
            <property name="text">
             <string>Тип диабета</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QComboBox" name="diabetesType">
            <property name="modelColumn">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="labelDisabilityType">
            <property name="text">
             <string>Инвалидность</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="disabilityType">
            <property name="modelColumn">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelVTEKNumber">
            <property name="text">
             <string>Номер ВТЭК</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QSpinBox" name="vtekNumber">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Срок действия</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QDateEdit" name="validity">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::UpDownArrows</enum>
            </property>
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="labelMedicine">
            <property name="text">
             <string>Лекарство</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="medicine"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="SearchTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Просмотр</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QScrollArea" name="scrollArea">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>172</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>577</width>
             <height>396</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QFormLayout" name="formLayout">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="labelDateAdd_2">
                <property name="text">
                 <string>Дата добавления</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QDateEdit" name="dateAddS0">
                  <property name="date">
                   <date>
                    <year>2001</year>
                    <month>1</month>
                    <day>1</day>
                   </date>
                  </property>
                  <property name="displayFormat">
                   <string>dd.MM.yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="dateAddS1">
                  <property name="date">
                   <date>
                    <year>2001</year>
                    <month>1</month>
                    <day>1</day>
                   </date>
                  </property>
                  <property name="displayFormat">
                   <string>dd.MM.yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelDistrict_2">
                <property name="text">
                 <string>Район</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="districtS">
                <property name="editable">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelIdNumber_2">
                <property name="text">
                 <string>Номер удостоверения</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QSpinBox" name="idNumberS0">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="idNumberS1">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                  <property name="value">
                   <number>10000000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelName_2">
                <property name="text">
                 <string>ФИО</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="nameS"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelBirthDate_2">
                <property name="text">
                 <string>День рождения</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QDateEdit" name="birthDateS0">
                  <property name="displayFormat">
                   <string>dd.MM.yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="birthDateS1">
                  <property name="displayFormat">
                   <string>dd.MM.yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="labelAddress_2">
                <property name="text">
                 <string>Адрес</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="addressS"/>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="labelPhoneNumber_2">
                <property name="text">
                 <string>Телефон</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="phoneNumberS"/>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="labelDiabetesType_2">
                <property name="text">
                 <string>Тип диабета</string>
                </property>
               </widget>
              </item>
              <item row="7" column="1">
               <widget class="QComboBox" name="diabetesTypeS"/>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="labelDisabilityType_2">
                <property name="text">
                 <string>Инвалидность</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QComboBox" name="disabilityTypeS"/>
              </item>
              <item row="9" column="0">
               <widget class="QLabel" name="labelVTEKNumber_2">
                <property name="text">
                 <string>Номер ВТЭК</string>
                </property>
               </widget>
              </item>
              <item row="9" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QSpinBox" name="vtekNumberS0">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="vtekNumberS1">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                  <property name="maximum">
                   <number>1000000000</number>
                  </property>
                  <property name="value">
                   <number>10000000</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="10" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Срок действия</string>
                </property>
               </widget>
              </item>
              <item row="10" column="1">
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QDateEdit" name="validityS0">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>—</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="validityS1">
                  <property name="buttonSymbols">
                   <enum>QAbstractSpinBox::NoButtons</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="11" column="0">
               <widget class="QLabel" name="labelMedicine_2">
                <property name="text">
                 <string>Лекарство</string>
                </property>
               </widget>
              </item>
              <item row="11" column="1">
               <widget class="QLineEdit" name="medicineS"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>591</width>
             <height>28</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="searchButton">
              <property name="text">
               <string>Поиск</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="showAll">
              <property name="text">
               <string>Показать все</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="exportButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Excel</string>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="saveButton">
              <property name="text">
               <string>Сохранить</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QTableView" name="clientTable">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QMenu" name="district_menu">
     <property name="title">
      <string>Районы</string>
     </property>
     <addaction name="add_district"/>
     <addaction name="edit_district"/>
    </widget>
    <widget class="QMenu" name="diabetesType_menu">
     <property name="title">
      <string>Типы диабета</string>
     </property>
     <addaction name="add_diabetesType"/>
     <addaction name="edit_diabetesType"/>
    </widget>
    <widget class="QMenu" name="disabilityType_menu">
     <property name="title">
      <string>Типы инвалидности</string>
     </property>
     <addaction name="add_disabilityType"/>
     <addaction name="edit_disabilityType"/>
    </widget>
    <addaction name="district_menu"/>
    <addaction name="diabetesType_menu"/>
    <addaction name="disabilityType_menu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="add_district">
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="edit_district">
   <property name="text">
    <string>Изменить/Удалить...</string>
   </property>
  </action>
  <action name="add_diabetesType">
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="edit_diabetesType">
   <property name="text">
    <string>Изменить/Удалить...</string>
   </property>
  </action>
  <action name="add_disabilityType">
   <property name="text">
    <string>Добавить...</string>
   </property>
  </action>
  <action name="edit_disabilityType">
   <property name="text">
    <string>Изменить/Удалить...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
